README.md

```diff
@@ -70,9 +70,9 @@ The rest of the code you write in React / Node.js / Prisma and just reference it
 
 <img width="800px" src="https://user-images.githubusercontent.com/1536647/231472285-126679e5-ecce-4cbb-8579-eb3cd9ba95bf.png"/>
 
-Given a simple .wasp configuration file that describes the high-level details of your web app, and .js(x)/.css/..., source files with your unique logic, Wasp compiler generates the full source of your web app in the target stack: front-end, back-end and deployment.
+Given a simple .wasp configuration file that describes the high-level details of your web app and .js(x)/.css/..., source files with your unique logic, the Wasp compiler generates the full source of your web app in the target stack: front-end, back-end, and deployment.
```

Comment:
This doesn't read right.
I think we should `,` right after `...`, not the one you removed.